lib/ansible/modules/system/ufw.py

```diff
@@ -303,13 +319,47 @@ def ufw_version():
         if command == 'state':
             states = {'enabled': 'enable', 'disabled': 'disable',
                       'reloaded': 'reload', 'reset': 'reset'}
-            execute(cmd + [['-f'], [states[value]]])
+            if module.check_mode:
+                if value in ['reloaded', 'reset']:
+                    changed = True
```

Comment:
The module currently doesn't return `changed == True` when called with `state == 'reloaded'` or `state == 'reset'` in non-check-mode. So check-mode should do the same and not set `changed = True` here.

(Except if you fix the behavior for non-check-mode as well :) )

Reply:
state reloaded and reset should have changed at true. I'll see I can do :)

Reply:
I would even argue that `reset` and `reload` always return `changed == True`, similar to `systemctl reload / restart`.

(I think `reload` simply recreates all ipfilter rules. That's probably useful if you screwed something up when editing them manually, or when you change some configuration, like enabling/disabling IPv6.)

Reply:
That's simpler so that find to me :)